src/feature-test/user_see_ingredient_after_add.test.js

```diff
@@ -0,0 +1,40 @@
+import '../setupTest.js'
+
+describe('I.I.A.S', () => {
+  it('User wants to see what they are about to submit', async () => {
+    const browser = await puppeteer.launch({
+      // This two lines below allows the dev to see the flow on the chromium page
+      // slowMo: 40,
+      // headless: false,
+      // devtools: true
```

Comment:
I thought we were removing all of these commented lines?

Reply:
I didn't get we wanted them to be deleted.
these comments are for the feature test if you want to see it in action while is testing. 
If they are not needed we can delete them, sure.